be/src/exec/pipeline/query_context.cpp

```diff
@@ -151,6 +151,17 @@ void QueryContext::init_mem_tracker(int64_t query_mem_limit, MemTracker* parent,
     });
 }
 
+MemTracker* QueryContext::operator_mem_tracker(int32_t plan_node_id) {
+    std::lock_guard<std::mutex> l(_operator_mem_trackers_lock);
+    auto it = _operator_mem_trackers.find(plan_node_id);
+    if (it != _operator_mem_trackers.end()) {
+        return it->second.get();
+    }
+    auto mem_tracker = std::make_shared<MemTracker>();
```

Comment:
Add name (such as plan_node_id or plan node name) to MemTracker,  will makes it easier to check the mem problem?